### PR TITLE
package.json: update @patternfly/react-core

### PR DESCRIPTION
**Pull request by croissanne**
Includes the `isPlain` styling for the `TextList` component.

### CODE REVIEW

File: src/Components/CreateImageWizard/formComponents/Packages.js [hunk 1]
```diff
@@ -275,7 +275,7 @@ const Packages = ({ getAllPackages, isSuccess }) => {
             ref={firstInputElement}
             onFocus={() => setFocus('available')}
             onBlur={() => setFocus('')}
-            onChange={(val) => setPackagesSearchName(val)}
+            onChange={(_, val) => setPackagesSearchName(val)}
```
[the remark this answers is not in the document]
Comment (croissanne) [reply src/Components/CreateImageWizard/formComponents/Packages.js:278]: Ah this is supposed to fix that problem, did you run npm ci after pulling?

[the remark this answers is not in the document]
Comment (regexowl) [reply src/Components/CreateImageWizard/formComponents/Packages.js:278]: Yeah, nevermind. Realized it about a minute ago, running the ci now :sweat_smile:

[the remark this answers is not in the document]
Comment (croissanne) [reply src/Components/CreateImageWizard/formComponents/Packages.js:278]: by this I mean the 2nd commit here

[the remark this answers is not in the document]
Comment (regexowl) [reply src/Components/CreateImageWizard/formComponents/Packages.js:278]: Yup, seems to be working well. Forgot to ci, sorry about that :sweat_smile:

**Review by regexowl — APPROVED** [review]

**Pull request merged**